src/structured-text/converters/convert_int_value.cpp

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************\
+
+Module: Structured Text Language Conversion
+
+Author: Matthias Weiss, matthias.weiss@diffblue.com
+
+\*******************************************************************/
+
+/// \file
+/// Structured Text Language Conversion
+
+#include "convert_int_value.h"
+#include <util/std_types.h>
+
+constant_exprt convert_int_value(const std::string &src)
+{
+  signedbv_typet type(16);
+  type.set(ID_structured_text_type, ID_structured_text_int_value);
```

Comment:
It looks like there should be a `structured_text_int_value_typet` or similar instead of reusing `signedbv_typet` and adding a flag

Reply:
Integers in STL are bitvectors, so they should be modelled this way. Both the Java and ANSI C parser use `signedbv_typet` in this fashion, e.g.:
```
signedbv_typet signed_int_type()
{
  signedbv_typet result(config.ansi_c.int_width);
  result.set(ID_C_c_type, ID_signed_int);
  return result;
}
```

However, I suggest creating an `stl_types.h` equivalent to `c_types.h` with:
```
#define STL_INT_WIDTH 16u

signedbv_typet stl_int_type()
{
  signedbv_typet result(STL_INT_WIDTH); 
  result.set(ID_structured_text_type, ID_structured_text_int_value); 
  return result;
}
```

And perhaps renaming `ID_structured_text_int_value` to just `ID_structured_text_int`.

Reply:
Renamed the IDs accordingly, I'd add `stl_types.h` in a future PR.